src/sentry_opsgenie/plugin.py

```diff
@@ -84,12 +84,7 @@ def build_payload(self, group, event):
             'entity': group.culprit,
         }
 
-        payload['details']['Event'] = dict(event.data or {})
-        
-        payload['tags'] = ','.join(
-            '%s:%s' % (str(k).replace(',', ''), str(v).replace(',', ''))
-            for k, v in dict(event.get_tags()).iteritems()
-        )
+        payload['tags'] = ['%s:%s' % (str(x).replace(',', ''), str(y).replace(',', '')) for x, y in event.get_tags()]
```

Comment:
I don't 100% understand the change in format here, were you able to send tags successfully like this?

Reply:
v2 API only accepts an array, not comma separated list. https://docs.opsgenie.com/docs/alert-api

Reply:
ah 👍